lookup/simple_env.go

```diff
@@ -3,14 +3,12 @@ package lookup
 import (
 	"fmt"
 	"os"
-
-	"github.com/docker/libcompose/project"
 )
 
 type OsEnvLookup struct {
 }
 
-func (o *OsEnvLookup) Lookup(key, serviceName string, config *project.ServiceConfig) []string {
+func (o *OsEnvLookup) Lookup(key string) []string {
```

Comment:
Ok, sounds fair enough :wink: I'll put them back (and add a little godoc on it :wink:)